app/services/eform_service.rb

```diff
@@ -1,47 +1,56 @@
 class EformService
-  SERVICES = sk.gov.schemas.servicebus.service._1.ServiceClassEnum
+  def initialize(proxy)
+    @upvs = proxy
+  end
 
-  attr_reader :object_factory
+  def self.const_missing(name)
+    sk.gov.schemas.servicebus.service._1.ServiceClassEnum.const_get(name)
```

Comment:
Co ja viem, neni toto zbytocna magia? Co bolo zle na tej konstante?

Reply:
toto sa mi videlo prilis dlhe a divne sa to citalo `EformService::SERVICES::` ako eForm sluzba - sluzby - ...